lib/rake/build.rake

```diff
@@ -4,14 +4,6 @@ require 'cdo/rake_utils'
 require 'cdo/git_utils'
 
 namespace :build do
-  desc 'Runs Chef Client to configure the OS environment.'
-  task :configure do
-    if CDO.chef_managed
-      HipChat.log 'Applying <b>chef</b> profile...'
-      RakeUtils.sudo 'chef-client'
-    end
-  end
-
```

Comment:
Rake-task change summary: The `chef-client`-run task is no longer included in `rake build`, instead it will be included in `ci:build` (via `ci:chef_update`), and only executed during the CI build process (which happens on daemon servers on update). For frontends, the `chef-client` run will now invoke `rake build`, rather than vice versa.